client/app/scripts/superdesk-publish/views/subscribers.html

```diff
@@ -0,0 +1,65 @@
+<div class="split-content">
+    <div class="header">
+        <h2 translate>Subscribers</h2>
+        <button class="btn btn-info pull-right" ng-click="edit()">
+            <i class="icon-plus-sign icon-white"></i> <span translate>Add New Subscriber</span>
+        </button>
+    </div>
+    
+    <div class="content">
+        <ul class="pills-list provider-list">
+            <li ng-repeat="subscriber in subscribers._items" class="clearfix">
+                <div class="header">
+                    <h6>{{ subscriber.name }}</h6>
```

Comment:
One-time binding here as well? (`subscriber.name` probably does not get changed here?)